start_bot_on_prod.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-cd daemon
-ENV="prod" go run . >> /var/log/pyrous-gobot.log 2>&1 &
+cd /srv/purous-gobot/daemon
```

Comment:
You can't call `cd`... you need `/bin/cd`

Reply:
Is cd not in the path for the daemon?